Source/WebKit/Shared/RTCNetwork.serialization.in

```diff
@@ -55,6 +55,7 @@ struct WebKit::RTCNetwork {
     Vector<WebKit::RTC::Network::InterfaceAddress> ips;
 };
 
+#if PLATFORM(COCOA) || PLATFORM(IOS_FAMILY)
```

Comment:
This is redundant.  It should be just PLATFORM(COCOA) which includes iOS family OSes.